mqtt/mqtt-edgehub/src/command.rs

```diff
@@ -90,14 +84,14 @@ impl CommandHandler {
             .map_err(CommandHandlerError::PollClientFailure)?
         {
             if let Err(e) = self.handle_event(event).await {
-                warn!(message = "failed to disconnect client", error = %e);
+                warn!(message = "error processing command handler event", error = %e);
```

Comment:
I made this more generic so if we want to add logic to the command handler we can